config/nrfconnect/chip-module/CMakeLists.txt

```
@@ -169,6 +169,10 @@ endif()
 
 if (CONFIG_CHIP_CRYPTO_PSA)
     matter_add_gn_arg_string("chip_crypto" "psa")
+    # PSA Spake2+ implementation is not ready for the POSIX target yet
+    if((NOT BOARD STREQUAL "native_posix") AND (NOT BOARD STREQUAL "native_posix_64"))
+        matter_add_gn_arg_bool  ("chip_crypto_psa_spake2p" TRUE)
+    endif()
```

```suggestion
    matter_add_gn_arg_bool  ("chip_crypto_psa_spake2p" NOT ARCH_POSIX)
```

or even:

```suggestion
    matter_add_gn_arg_bool  ("chip_crypto_psa_spake2p" CONFIG_PSA_WANT_ALG_SPAKE2P_MATTER)
```

just in case SPAKE2P is not available on some crypto configuration.